en/getting_help/can_i_use_jquery_or_other_dom_manipulating_js.md

```diff
@@ -21,10 +26,17 @@ class PostController < Volt::ModelController
   end
 end
 ```
+This uses jQuery to access an element by known class. This can work well if you have few such cases
```

Comment:
I think we're better off if we just don't mention doing $('.some-class'), since most of the time thats not going to work if a component gets used in more than one place.